### ISSUE TITLE
What fonts program uses for GUI?

### ISSUE THREAD

**Issue by Seamas73** [SAMMISolutions/SAMMI-Official#11]
I tried to translate the program interface into Russian, but on some (not all) buttons I see only rectangles instead of letters. Apparently, some font is missing in the system, and it must be found and installed, or some other font present in the system must be substituted instead. But for this you need to know what fonts the application uses, and see if I have them installed.
So, the question is: what fonts are used in your program to display the interface?

**Comment by SebasF1349** [comment]
Hi @Seamas73 If some characters are missing we need to add them to SAMMI. Which buttons didn't render properly?

Also, did you join our discord server https://discord.gg/3RmMXY4E? If you want to share your Russian translation with other users you can contact me there.

**Comment by Seamas73** [comment]
I just started translating (translated "Language", "Button Apperance" and "Menu" section of language ini-file and began translation of "Twitch Connections" section). I began to check whether the translated words get out of the boundaries of the interface elements, and saw rectangles except letters in some buttons and menu points. I'm attaching screenshots of interface elements for comparison.

![2022-11-18_12-46-25 (2)](https://user-images.githubusercontent.com/36772070/202672842-ebb2b194-f59c-4a3a-b2cf-65bbab087bfe.png)
![2022-11-18_12-44-34 (2)](https://user-images.githubusercontent.com/36772070/202672853-0d18b66c-46a6-469a-93b8-993759bb41fa.png)
![2022-11-18_12-45-39 (2)](https://user-images.githubusercontent.com/36772070/202672857-7fe6e1f7-9601-47dd-a471-9ef8f2d72efc.png)
![2022-11-18_12-43-48 (2)](https://user-images.githubusercontent.com/36772070/202672860-21f7a356-d2c5-4e44-9c65-45a4efb8d0f7.png)

**Comment by Seamas73** [comment]
Translated "About Window" section:
![2022-11-18_13-22-54](https://user-images.githubusercontent.com/36772070/202680161-01fa514e-8be2-4707-b156-d5d2ee0a019e.png)
![2022-11-18_13-21-15](https://user-images.githubusercontent.com/36772070/202680166-08ad656f-19d8-4a6c-8efa-c384b8a123d0.png)

**Comment by Seamas73** [comment]
Translated "VersionCheck" section:

![2022-11-18_14-06-25](https://user-images.githubusercontent.com/36772070/202691760-a47b2bd6-523f-43c0-b7db-47cd6d735df4.png)
![2022-11-18_14-05-37](https://user-images.githubusercontent.com/36772070/202691768-b2efd94c-cfab-4a10-91f4-7bc876ff433f.png)
![2022-11-18_14-05-05](https://user-images.githubusercontent.com/36772070/202691770-4154c14b-9c1f-4ed7-92cf-a5e207d1bce2.png)
![2022-11-18_14-04-05](https://user-images.githubusercontent.com/36772070/202691776-51fecd59-4207-4d57-9442-2610af53de09.png)

Without fixing problem with fonts further translation is to be difficult: I need to see if my translation get out of the boundaries of the interface elements, to find the way to avoid it.

**Comment by SebasF1349** [comment]
Thanks for the screenshots. We will fix it, hope we can do it for the next release.

**Comment by SebasF1349** [comment]
@Seamas73 the new version released today should fix this issue.

**Comment by Seamas73** [comment]
Yes. Now in those windows, the text in which I have already translated, everything is readable, no rectangles.

**Comment by Seamas73** [comment]
Wen I start SAMMY deck and load any deck I see rectangles on buttons except cyrillic letters
![изображение_2023-01-13_181338114](https://user-images.githubusercontent.com/36772070/212353747-7abff2e9-6b16-4f14-a84c-83e0527370f8.png)

**Comment by Seamas73** [comment]
> When I start SAMMY deck and load any deck I see rectangles on buttons except cyrillic letters ![изображение_2023-01-13_181338114](https://user-images.githubusercontent.com/36772070/212353747-7abff2e9-6b16-4f14-a84c-83e0527370f8.png)

On android stream deck I see rectangles on buttons except cyrillic letters too
![IMG_20230114_120746](https://user-images.githubusercontent.com/36772070/212464769-2d634dfb-aa88-4ca7-9a21-432c220f1e78.jpg)

**Comment by Seamas73** [comment]
In popup system messages generated by SAMMI, I see unreadable symbols instead of cyrillic letters
![2023-01-14_13-26-33](https://user-images.githubusercontent.com/36772070/212467673-e5ab8d82-0647-4d6e-afac-aabe72e2e933.png)

**Comment by Seamas73** [comment]
Can you fix fonts issue for SAMMI Deck and Android version of SAMMI Deck?

**Comment by christinna9031** [comment]
Please submit a bug report in our Discord channel if this is still an issue even with the new SAMMI.dll.
